src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Class names should not contain verbs. If 'Figure' is not an appropriate name, consider renaming it to something like 'Shape' or 'GeometricFigure'.